checks/container.go

```diff
@@ -139,15 +130,26 @@ func fmtContainers(
 			Started:     ctr.StartedAt,
 			Tags:        tags,
 		})
+	}
+	return containers
+}
+
+// chunkContainers formats and chunks the ctrList into a slice of chunks using a specific number of chunks.
+func chunkContainers(ctrList []*containers.Container, lastRates map[string]util.ContainerRateMetrics, lastRun time.Time, chunks, perChunk int) [][]*model.Container {
+	chunked := make([][]*model.Container, 0, chunks)
+	chunk := make([]*model.Container, 0, perChunk)
```

Comment:
It seems redundant to pass both perChunk and chunk because `perChunk * chunks ~= len(ctrList)`. Can we get away with only passing one?

Reply:
I did this because before calling `chunkContainers` the calculation is already done once: https://github.com/DataDog/datadog-process-agent/pull/215/files#diff-69c4a56eada3efac057ba71ef68a0aa2R60, it's just a matter of passing additional parameter or do the calculation twice.